apps/wakunode2/networks_config.nim

```diff
@@ -25,6 +27,9 @@ proc TheWakuNetworkConf*(T: type ClusterConf): ClusterConf =
     rlnRelayEthContractAddress: "0xF471d71E9b1455bBF4b85d475afb9BB0954A29c4",
     rlnRelayDynamic: true,
     rlnRelayBandwidthThreshold: 0,
+    rlnEpochSizeSec: 1,
+    # parameter to be defined with rln_v2
+    rlnRelayUserMessageLimit: 1,
```

Comment:
this belongs to rln_v2 but if I used the `when defined(rln_v2)` I have to duplicate everything. So adding it directly here.